pom.xml

```diff
@@ -2662,13 +2662,13 @@
         <jetty.wso2.version>8.1.17.v20150415.wso2v1</jetty.wso2.version>
         <libthrift.version>0.16.0.wso2v1</libthrift.version>
         <opencsv.version>2.3</opencsv.version>
-        <fasterxml.jackson.version>2.14.1</fasterxml.jackson.version>
+        <fasterxml.jackson.version>2.16.1</fasterxml.jackson.version>
         <json.simple.version>1.1.1</json.simple.version>
         <esotericsoftware.kryo.version>3.0.3</esotericsoftware.kryo.version>
         <org.objenesis.version>2.1</org.objenesis.version>
         <esotericsoftware.minlog.version>1.3.0</esotericsoftware.minlog.version>
-        <orbit.version.snakeyaml>1.33.0.wso2v1</orbit.version.snakeyaml>
-        <orbit.snakeyaml.import.version.range>[1.16.0.wso2v1, 2.0.0)</orbit.snakeyaml.import.version.range>
+        <orbit.version.snakeyaml>2.0</orbit.version.snakeyaml>
```

Comment:
```suggestion
        <orbit.version.snakeyaml>2.2</orbit.version.snakeyaml>
```

Reply:
Fixed with: https://github.com/wso2/carbon-mediation/pull/1695/commits/7c4f129069eb08350f6eccc11855619ff611d1ce.
Thank you.